Bambu/source/FatJetsFiller.cc

```diff
@@ -21,9 +21,28 @@ mithep::nero::FatJetsFiller::defineBranches(TTree* _tree)
 }
 
 void
+mithep::nero::FatJetsFiller::initialize()
+{
+	topANN = new NeuralNet(5,2);
+	#include "topTagger_simple.icc" // these are just weights - move to MIT_DATA?
```

Comment:
My intention of saving it as C++ object was that it would be portable. I'm not sure how much use plain text or xml would be, since for more complicated NNs, looking at the weights won't give you much insight. I can always add a reader to the class, however.